tools/ka-customizer-runner

```diff
@@ -1,11 +1,14 @@
 #!/bin/bash -ex
 
+BRANCH=$1
+shift
+
 if [ "$1" = "update" ]; then
     shift
-    flatpak update com.endlessm.Platform master
-    flatpak update com.endlessm.Sdk master
+    flatpak update "com.endlessm.Platform//$BRANCH"
+    flatpak update "com.endlessm.Sdk//$BRANCH"
```

Comment:
I got an error when running the ka-customizer:

```
+ BRANCH=
+ shift
+ '[' update = update ']'
+ shift
+ flatpak update com.endlessm.Platform//
Looking for updates...
Updating: com.endlessm.Platform/x86_64/eos3.1 from eos-runtimes
./ka-customizer-runner: line 11:  1703 Segmentation fault      (core dumped) flatpak update "com.endlessm.Platform//$BRANCH"
```

The segmentation fault part is probably unrelated, but I think the BRANCH is not getting replaced?